co2eq/meal.js

```diff
@@ -31,8 +32,8 @@ export const explanation = {
 
 export const modelCanRunVersion = 1;
 export function modelCanRun(activity) {
-  const { mealType, ingredients } = activity;
-  if (mealType || (ingredients && Object.keys(ingredients).length > 0)) {
+  const { mealType, lineItems } = activity;
+  if (mealType || (lineItems && lineItems.length && lineItems.filter(l => l.unit === UNIT_KILOGRAMS).length)) {
```

Comment:
What happens if one line is a meal and the other is a hotel stay? The mixed type will not work.
Maybe this model should only run where all lineItems are ingredients?

Reply:
yep, makes sense to check this. I have added the check. Just to confirm: in the _current_ state of the models, we should not have a mixed type, right?

Reply:
if it's a purchase, then we can't have a mixed type
if it's a meal, then we also can't have mixed types (we have mixed ingredient but the purchaseType is actually not even relevant here as the activity type is meal)